docs/dev/resources/networking.md

```diff
@@ -0,0 +1,57 @@
+# Networking Common Tasks and Information
+
+## Quick Access
+[Allow Connection to Sites on the Internet from a ProB workload](#allowing-connection-to-sites-on-the-internet)
+
+[Allow Connection to internal application](#allow-connection-to-internal-application)
+
+[DNS pitfalls](#dns-pitfalls-to-internal-application-connecting)
+
+[The `Priority` field in the fwprcg](#the-priority-field)
+
+[In which repo should we define our resources?](#in-which-repository-should-we-define-the-resources)
+
+----
+### Allowing Connection to sites on the Internet
+This is a bit more niche, as our unclassified notebooks already have connection to the internet, but this can come up when we have a Protected B certified application like seen in https://github.com/StatCan/aaw-private/issues/125 where it's accessible to the internet, but unaccecssible from our Protected B workloads.

```

Comment:
Yes whitelisting of internet domains is applicable to protected-b and system nodes, since unclassified is open by default.